### PR TITLE
Update environment vars to wharf-api v5 form

### LINKED ISSUE #12: Update environment variables
We had backwards compatibility for the old format for a while, but this will be removed from wharf-api in v5.0.0 (iver-wharf/wharf-api#87).

Following wharf-api v5.0.0, all wharf process-related environment variables available through the new config system need to be prefixed with `WHARF_` in the `docker-compose.yml` file, and some of the names have changed slightly.

The name for each variable can be deduced from the structure in the `config.go` file, e.g.:
```go
type Config struct {
	DB   DBConfig
}

type DBConfig struct {
	Password string
}
```

The appropriate env var is then `WHARF_DB_PASSWORD`.
I.e., it is written the way you would access it from the config in Go, `cfg.DB.Password`, after performing the following steps:
1. Convert to uppercase -> `CFG.DB.PASSWORD`
2. Replace `CFG` with `WHARF` -> `WHARF.DB.PASSWORD`
3. Replace all `.` with `_` -> `WHARF_DB_PASSWORD`

### PR DESCRIPTION
**Pull request by fredx30**
## Summary

Changes environment variables to follow the form introduced with config files for the v5 wharf-api.

This also adds a variable to set usage of the postgresql drvier. Without this the docker-compose file becomes uncohesive- the database postgres is brought online and used. 

The file here https://github.com/iver-wharf/wharf-api/blob/master/config.go is a good guide for checking the names suggested.

## Motivation

I should make it easier to deduce the alternative options when need comes to change the file for development.
It also allows deprecation of the older form.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,29 +13,24 @@ services:
     - "5000:8081"
     volumes:
     - ./wharf-docker-compose/traefik:/etc/traefik # Traefik configuration file
-  web:
-    image: quay.io/iver-wharf/wharf-web
+#  web:
+#    image: w-web:1
+#    ports:
+#    - "8080:8080"
```
Comment (Alexamakans) [inline docker-compose.yml]: This probably shouldn't be here, right?
```suggestion
  web:
    image: quay.io/iver-wharf/wharf-web
```

**Review by Alexamakans — CHANGES_REQUESTED** [review]
Nice 👍🏻

File: docker-compose.yml (hunk 1 above)
Comment (fredx30) [reply docker-compose.yml]: I think maybe your right.  My crazy debugging stuff will be removed.
We have had the web in there commented out though. Keeping it there commented makes sense with the documentation. 
It may make sense to rethink the port on the web to to satisfy both the requirement of not colliding it with the baremetal `npm run start` and also keeping in compliance with the defaults for CORS when using OIDC. 
My sugestion here would be port **4201** thoughts?

File: docker-compose.build.yml [hunk 2]
```diff
@@ -10,8 +10,8 @@ services:
     ## Slight warning that it does take significantly more time to build than
     ## the other services though.
     deploy:
-      replicas: 0
-    #build: wharf-web
```
Comment (Alexamakans) [inline docker-compose.build.yml:14]: What is the motivation for this change?

File: docker-compose.build.yml [hunk 3]
```diff
@@ -10,8 +10,8 @@ services:
...
+      replicas: 1
+    build: wharf-web
```
Comment (Alexamakans) [inline docker-compose.build.yml]: What is the motivation for this change?

Comment (Alexamakans) [reply docker-compose.build.yml]: Oops, I forgot this was a draft and that it's probably part of testing?

File: docker-compose.yml [hunk 4]
```diff
@@ -22,20 +22,13 @@ services:
...
-      RABBITMQNAME: wharf_queue
-      RABBITMQDISABLESSL: "true"
-      RABBITMQCONNATTEMPTS: 10
+      WHARF_HTTP_DB_DRIVER: postgres
+      WHARF_HTTP_DB_HOST: db
+      WHARF_HTTP_DB_PORT: 5432
+      WHARF_HTTP_DB_USERNAME: postgres
+      WHARF_HTTP_DB_PASSWORD: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
+      WHARF_HTTP_DB_NAME: wharf
+      WHARF_HTTP_DB_LOG: "true"
```
Comment (Alexamakans) [inline docker-compose.yml]: In trying it out I realized these are all wrong. Shouldn't have `_HTTP`
```suggestion
      WHARF_DB_DRIVER: postgres
      WHARF_DB_HOST: db
      WHARF_DB_PORT: 5432
      WHARF_DB_USERNAME: postgres
      WHARF_DB_PASSWORD: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
      WHARF_DB_NAME: wharf
      WHARF_DB_LOG: "true"
```

**Review by Alexamakans — CHANGES_REQUESTED** [review]

**Comment by fredx30** [comment]
@jilleJr, Hello!
Im handing this over to you. good luck with my nasty code :)

File: docker-compose.build.yml (hunk 3 above)
Comment (applejag) [reply docker-compose.build.yml]: Reverted.

**Review by Alexamakans — APPROVED** [review]
